test/e2e/specs/footer/00-footer.spec.js

```diff
@@ -0,0 +1,61 @@
+var chaisePage = require('./../../utils/chaise.page.js');
+var chaiseConfig = {
+    name: "Footer",
+    apps:['record','recordedit','recordset'],
+    defaultTables: {
+        "1": {
+            "schema": "isa",
+            "table": "dataset",
+            "id": "269"
+        }
+    }
+};
+var EC = protractor.ExpectedConditions;
+describe('Page Footer', function() {
+            chaiseConfig.apps.forEach(function(val){
+
+                let flocation = "/" + val + "/#1/" + chaiseConfig.defaultTables[1].schema + ":" + chaiseConfig.defaultTables[1].table + "/id=" + chaiseConfig.defaultTables[1].id;
```

Comment:
Use a schema that will be created on the fly with test framework. These schema and table names  that you have used here might change and this test case might break.